src/pipit/settings/prod.py

```diff
@@ -74,6 +71,11 @@
 # Use a secure cookie for the CSRF cookie
 CSRF_COOKIE_SECURE = True
 
+# Sentry
+SENTRY_DSN = get_env("SENTRY_DSN", "")
+SENTRY_ENVIRONMENT = get_env("SENTRY_ENVIRONMENT", "production")
```

Comment:
Should we assume that only the production could fallback on the default arg? Perhaps add an argument to `docker compose` files for cases that can mimic production behavior (like pizzaoven)

Reply:
Added `SENTRY_ENVIRONMENT=pizzaoven`, to python.env also on the server.